src/aitsi/m3spin/pkb/interfaces/Parent.java

```diff
@@ -8,6 +8,8 @@ public interface Parent {
 
     List<TNode> getParentedBy(TNode p);
```

Comment:
Skoro relacja Parent dotyczy wyłącznie instrukcji, to zastanawiam się, czy nie zastąpić wszystkich typów `TNode` w tym interfejsie typem `Statement`

Reply:
Zrobion i pogrzebion